spec/bundler/env_spec.rb

```diff
@@ -139,5 +139,9 @@
         expect(described_class.report).to include("Git         1.2.3 (Apple Git-BS)")
       end
     end
+
+    it "properly parses version of tools when shelling out to them" do
+      expect(described_class.send(:version_of, "ruby")).to_not include("\n")
```

Comment:
For the test I thought I could shell out directly to `ruby` since we know for sure it will be in the path?